core/views.py

```diff
@@ -130,4 +130,26 @@ def post(self, request, *args, **kwargs):
         except Exception as e:
             pass
         
-        return redirect(self.success_url)
\ No newline at end of file
+        return redirect(self.success_url)
+
+
+
+class AboutView(View):
```

Comment:
Please use `[TemplateView](https://docs.djangoproject.com/en/3.1/ref/class-based-views/base/#django.views.generic.base.TemplateView)` instead of `View`